source/openpulse/README.md

```diff
@@ -0,0 +1,32 @@
+# OpenPulse Python Reference
+
+[![License](https://img.shields.io/github/license/openqasm/openqasm.svg)](https://opensource.org/licenses/Apache-2.0)<!-- long-description-skip-begin -->[![Release](https://img.shields.io/pypi/v/openpulse)](https://pypi.org/project/openpulse)<!-- long-description-skip-end -->
+
+OpenPulse is a minor extension to OpenQASM 3 with a concrete implementation of pulse grammar.
```

Comment:
Can we add a TODO note here about refactoring the grammar as @jakelishman has suggested?

Reply:
Added TODO to README.